### PR TITLE
feat: update slack

### PR DESCRIPTION
**Pull request by labbomb**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by SbloodyS — APPROVED** [review]
LGTM

File: docs/configs/site.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default {
   'en-us': {
     banner: {
       text: '🤔 Have queries regarding Apache DolphinScheduler, Join Slack channel to disscuss them ',
-      link: 'https://join.slack.com/t/asf-dolphinscheduler/shared_invite/zt-1e0s4agia-Ofkxa93zStmqTm9UsQBE8g'
+      link: 'https://join.slack.com/t/asf-dolphinscheduler/shared_invite/zt-1e36toy4n-5n9U2R__FDM05R~MJFFVBg'
```
Comment (ruanwenjun) [inline docs/configs/site.js:33]: Why we need to update the slack URL? I find the before link can work well.

Comment (davidzollo) [reply docs/configs/site.js:33]: > Why we need to update the slack URL? I find the before link can work well.

@labbomb I have the same question, is the invitation url no valid?

Comment (caishunfeng) [reply docs/configs/site.js:33]: PTAL @nielifeng

Comment (EricGao888) [reply docs/configs/site.js:33]: Is there a 30-day expiration for the invitation link?

Comment (nielifeng) [reply docs/configs/site.js:33]: Change account and regenerate link

Comment (labbomb) [reply docs/configs/site.js:33]: So if it doesn't seem to be a problem, this PR is merged, right

**Review by nielifeng — APPROVED** [review]

**Pull request merged**